x/auction/types/params.go

```diff
@@ -0,0 +1,56 @@
+package types
+
+import (
+	"fmt"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// Parameter keys
+var (
+	KeyPriceMaxBlockAge = []byte("pricemaxblockage")
+)
+
+var _ paramtypes.ParamSet = &Params{}
+
+// ParamKeyTable returns the parameter key table.
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+// DefaultParams returns default oracle parameters
+func DefaultParams() Params {
+	return Params{
+		PriceMaxBlockAge: 201600, // roughly two weeks based on 6 second blocks
```

Comment:
Every 2 weeks might be a bit too frequent to keep submitting token price governance updates and rallying votes

Reply:
send to 4 weeks